proxy/src/main/java/com/wavefront/agent/listeners/tracing/JaegerThriftCollectorHandler.java

```diff
@@ -345,7 +346,7 @@ private void processSpan(io.jaegertracing.thriftjava.Span span,
     }
     Span wavefrontSpan = Span.newBuilder()
         .setCustomer("dummy")
-        .setName(span.getOperationName())
+        .setName(sanitizeValue(span.getOperationName()))
```

Comment:
sanitizeValue returns a value wrapped in quotes so it's going to ruin the span name with adding extraneous quotes. We should fix sanitizeValue.